client/src/document/index.scss

```
@@ -7,6 +7,8 @@
 @import "~@mdn/minimalist/sass/mixins/utils";
 
 @import "./molecules/breadcrumb-locale-container";
+@import "./molecules/notecards";
+@import "./molecules/localized-content-note/";
```

```suggestion
@import "./molecules/localized-content-note";
```
more consistent I suppose.